build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
   ext.junitJupiterVersion = '5.6.1'
-  ext.pegasusVersion = '28.3.7'
+  ext.pegasusVersion = '29.22.16'
```

Comment:
This was bumped to support gradle 6, the old version did not support it. Decided to conservatively increase this version to what appeared to be the latest stable 29.x without moving to the very latest 30.x. Didn't want to move too fast here.